ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -575,6 +580,26 @@ import { DraggableListBehavior } from '../nuxeo-draggable-list-behavior.js';
       });
     }
 
+    _computeActionsStyle(editable, orderable) {
+      afterNextRender(this, () => this._alignHeaderRow());
```

Comment:
Note that `_computeActionsStyle` will be called twice, which means `afterNextRender(this, () => this._alignHeaderRow());` will be called one more time than it's needed. We could put this line inside an observer, to mitigate the problem.

Also, **usually** computed bindings are pure functions: they return a value but they do not affect the element's state directly.

Reply:
Right, thanks for the observation. I will refactor de code.